### DISCUSSION TITLE
Thank you, once more.

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#8895] [appreciation]
Not only I'm proud of announcing that I've finally won a chess game after 2 years, but I'm happy for another update for Monii!
Thank you. Keep it up! 

P.S: This chess round lasted more than 30 min. XD

![20210216230225_1](https://user-images.githubusercontent.com/48300362/108275588-76e51d00-716e-11eb-881b-319115d7a77d.jpg)

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
